test/integration/node-specific/operation_example.test.js

```diff
@@ -4732,13 +4696,15 @@ describe('Operation Examples', function () {
           var cursor = collection.find({});
           // Fetch the first object off the cursor
           cursor.next(function (err, item) {
-            test.equal(0, item.a);
+            expect(err).to.not.exist;
```

Comment:
This one tests was failing and you don't get to see the real error because it's in `err` which wasn't being asserted. So I have to add err exists check and I fixed the other assertion while I was here. We've done drive by improvements like that, at the very least we need to be asserting the error does not exist.

Reply:
Sounds good 👍